### ISSUE TITLE
Microsoft's go-winmd parser

### ISSUE THREAD

**Issue by mdepot** [zzl/go-winapi-gen#1]
This project currently uses it's own parser for winmd files (https://github.com/zzl/go-winmd), which pre-dates (on GitHub at least) a Go winmd parser published by Microsoft (https://github.com/microsoft/go-winmd).

Would using Microsoft's own parser provide any advantage to this project?

**Comment by zzl** [comment]
Yes I'm aware of the go-winmd project from Microsoft, I'll look into it when it's been officially released.
